hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/WALPlayer.java

```diff
@@ -225,8 +245,10 @@ public void map(WALKey key, WALEdit value, Context context)
                 }
               }
               if (CellUtil.isDelete(cell)) {
+                context.getCounter(Counter.CELLS_WRITTEN).increment(1);
                 del.add(cell);
               } else {
+                context.getCounter(Counter.CELLS_WRITTEN).increment(1);
                 put.add(cell);
```

Comment:
Nit, can the above "context.getCounter(Counter.CELLS_WRITTEN).increment(1);" be moved of the if {} else {}?

Reply:
Makes sense.